### PR TITLE
Documentation for R4 Immunization Update

### PR DESCRIPTION
**Pull request by Likitha-Seeram**
Adding Documentation for Immunization update.

### CODE REVIEW

File: lib/resources/r4/immunization.yaml [hunk 1]
```diff
@@ -27,6 +52,18 @@ fields:
...
+  cardinality: 1..1
+  type: code
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.status
+  description: Indicates the current status of the immunization event.
+  action: update
+  example: |
+    {
+      "status": "not-done"
+    }
+
```
Comment (MaxPhilips) [inline lib/resources/r4/immunization.yaml]: for all these duped fields, you should instead be able to do this:
```
  action:
    - create
    - update
```

File: content/millennium/r4/medications/immunization.md [hunk 2]
```diff
@@ -166,6 +166,58 @@ The `ETag` response header indicates the current `If-Match` version to use on su
...
+    PUT /Immunization/:id
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true, patient: false) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/json+fhir',
+                   'Content-Type': 'application/json+fhir', 'If-Match': 'W/"&lt;Current version of the Immunization resource>"'} %>
```
Comment (MaxPhilips) [inline content/millennium/r4/medications/immunization.md]: can remove Accept, and switch content-type to application/fhir+json

File: lib/resources/r4/immunization.yaml [hunk 3]
```diff
@@ -15,6 +15,31 @@ fields:
...
+  action: update
+  example: |
+    {
+      "resourceType": "Immunization"
+    }
+  note: resourceType must be Immunization
+
+- name: id
+  required: 'Yes'
+  cardinality: 0..1
```
Comment (MaxPhilips) [inline lib/resources/r4/immunization.yaml]: id should have cardinality 1..1 indicating it's required

File: lib/resources/r4/immunization.yaml (hunk 1 above)
Comment (Likitha-Seeram) [reply lib/resources/r4/immunization.yaml]: Updated all the fields. https://github.com/cerner/fhir.cerner.com/pull/295/commits/fe9f32d2c48645c5ba462f17793818a3e32b5d4a#diff-2c50e8f41daaba3a640861e7b6e81488L11

File: content/millennium/r4/medications/immunization.md (hunk 2 above)
Comment (Likitha-Seeram) [reply content/millennium/r4/medications/immunization.md]: Updated. https://github.com/cerner/fhir.cerner.com/pull/295/commits/fe9f32d2c48645c5ba462f17793818a3e32b5d4a#diff-3e96067fd99a2fee85880712d23a6719R181

File: lib/resources/r4/immunization.yaml (hunk 3 above)
Comment (Likitha-Seeram) [reply lib/resources/r4/immunization.yaml]: Updated it. https://github.com/cerner/fhir.cerner.com/pull/295/commits/fe9f32d2c48645c5ba462f17793818a3e32b5d4a#diff-2c50e8f41daaba3a640861e7b6e81488R22

**Comment by MaxPhilips** [comment]
this PR is into the immunization-write branch, so we will merge #276 then rebase this on master

**Comment by MaxPhilips** [comment]
post-deploy validation

Screen Shot 2020-01-29 at 3 36 15 PM

**Pull request merged**